File: Task Tracker Java/contest_java/contestDomain/src/main/java/com/example/dto/ParticipantDTO.java
package com.example.dto;

import java.io.Serializable;
import java.util.Objects;

public class ParticipantDTO implements Serializable {
    private String name;
    private int age;

    public ParticipantDTO(String name, int age) {
        this.name = name;
        this.age = age;
    }

    public String getName() {
        return name;
    }

    public int getAge() {
        return age;
    }

    @Override
    public boolean equals(Object o) {
        if (this == o) return true;
        if (o == null || getClass() != o.getClass()) return false;
        ParticipantDTO that = (ParticipantDTO) o;
        return age == that.age && Objects.equals(name, that.name);
    }

    @Override
    public int hashCode() {
        return Objects.hash(name, age);
    }
}

File: Task Tracker C#/contest/contestDomain/dto/ParticipantDTO.cs
namespace contestDomain.dto
{
    [Serializable]
    public class ParticipantDTO
    {
        public string name { get; set; }
        public int age { get; set; }

        public ParticipantDTO(string name, int age)
        {
            this.name = name;
            this.age = age;
        }
    }
}
